docs/monoDrive_home/Viewport.md

```diff
@@ -5,23 +5,36 @@ the monoDrive Simulator or Scenario Editor.
 
 ## Configuration
 
-```
-[
-  {
-    "type": "ViewportCamera",
-    "location": {
-        "x": 0.0,
-        "y": 0.0,
-        "z": 250.0
-      },
-      "rotation": {
-        "pitch": 0.0,
-        "yaw": 0.0,
-        "roll": 0.0
-      },
-      "fov": 60.0,
+``` json
+{
+  "type": "ViewportCamera",
+  "channel_depth": 1,
+  "channels": "bgra",
+  "dynamic_range": 50.0,
+  "focal_length": 9.0,
+  "fov": 60.0,
+  "fstop": 1.39999997615814,
+  "listen_port": 0,
+  "location": {
+      "x": 0.0,
+      "y": 0.0,
+      "z": 0.0
+  },
+  "max_distance": 50000.0,
+  "max_shutter": 0.00139999995008111,
+  "min_shutter": 0.000500000023748726,
+  "rotation": {
+      "pitch": 0.0,
+      "roll": 0.0,
+      "yaw": 0.0
+  },
+  "sensor_size": 9.06999969482422,
+  "stream_dimensions": {
+      "x": 512,
+      "y": 512
```

Comment:
The old configuration was more accurate. This sensor inherits from the camera sensor and therefore has all the same variables. However, for the viewport most of these are just ignored.

Reply:
This still needs be paired down to only the necessary variables.